pkg/scheduler/internal/queue/scheduling_queue.go

```diff
@@ -59,7 +59,8 @@ const unschedulableQTimeInterval = 60 * time.Second
 type SchedulingQueue interface {
 	Add(pod *v1.Pod) error
 	AddIfNotPresent(pod *v1.Pod) error
-	AddUnschedulableIfNotPresent(pod *v1.Pod) error
+	AddUnschedulableIfNotPresent(pod *v1.Pod, podSchedulingCycle int64) error
+	SchedulingCycle() int64
```

Comment:
nits: one last request is could you please add a comment to explain this `SchedulingCycle` interface a bit? It would be helpful to upcoming contributors.

Reply:
Also, a comment for `AddUnschedulableIfNotPresent` to explain `podSchedulingCycle int64`.

Reply:
Comments are added. PTAL.